api/database/schema_create.sql

```diff
@@ -0,0 +1,23 @@
+CREATE DATABASE  IF NOT EXISTS `protectplayanow2` /*!40100 DEFAULT CHARACTER SET utf8 */;
+USE `protectplayanow2`;
+
+DROP TABLE IF EXISTS `reading`;
```

Comment:
Doesn't the framework we use automatically create the DB tables corresponding to the MVC models?

Reply:
it's generally frowned upon to have autogenerated schemas. it's ok for tests or examples but not for 'prod' systems.

Reply:
First time I hear of that. We did use them in production at Amazon RDS.

Reply:
@pgan002 this issue is that one has a higher risk of data loss vs the other